m46x/StdDriver/drv_emac/synopGMAC_Dev.c

```diff
@@ -0,0 +1,2947 @@
+/* ===================================================================================
+ * Copyright (c) <2009> Synopsys, Inc.
+ *
```

Comment:
Apache-2.0 license please

Reply:
Thanks of your kindly review.
Fixed in commit https://github.com/zephyrproject-rtos/hal_nuvoton/commit/17e131d6cf28744a149250e72113611fbe278370